media-gfx/imagemagick/imagemagick-6.9.6.4.recipe

```diff
@@ -8,10 +8,10 @@ draw text, lines, polygons, ellipses and Bézier curves."
 HOMEPAGE="http://www.imagemagick.org"
```

Comment:
Could you please change this to:
```
HOMEPAGE="https://www.imagemagick.org/"
```
(Please don't forget the trailing slash ;-)
Thanks!